package.json

```diff
@@ -39,21 +38,14 @@
     "@babel/plugin-transform-runtime": "7.15.0",
     "@babel/preset-env": "7.15.0",
     "@trivago/prettier-plugin-sort-imports": "2.0.2",
-    "@types/jest": "^24.0.24",
-    "@types/node": "^12.12.21",
-    "@types/puppeteer": "^2.0.0",
-    "@types/signale": "^1.2.1",
     "eslint": "7.32.0",
     "install-peers-cli": "^2.2.0",
     "jest": "27.0.6",
-    "jest-puppeteer": "^4.4.0",
-    "puppeteer": "^2.0.0",
-    "ts-jest": "^25.3.1",
-    "tsc-watch": "^4.0.0",
-    "typescript": "^3.7.4"
+    "jest-puppeteer": "5.0.4",
+    "puppeteer": "10.2.0"
   },
   "peerDependencies": {
-    "puppeteer": "^2.0.0"
+    "puppeteer": "10.2.0"
   },
   "optionalDependencies": {
     "puppeteer-extra-plugin-adblocker": "^2.11.1",
```

Comment:
Let's remove all these optional dependencies too.
Everything related to `puppeteer-extra`

Reply:
Done.